core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -694,6 +694,22 @@ class Ruleset {
                     RulesetErrorSeverity.Warning
                 )
             }
+            for (unique in improvement.uniqueObjects) {
+                if (unique.type == UniqueType.PillageYieldRandom || unique.type == UniqueType.PillageYieldFixed) {
+                    val params = Stats.parse(unique.params[0])
```

Comment:
We're in the mod checker. Stats.parse will throw (null pointer no less) if fed garbage. Better Stats.isStats first.

Reply:
Good call.

Reply:
But... but... that's an Error! Imagine finding `Pillaging this improvement yields [42 Petunias]` and _not_ nagging about it! Even if they `continue` to fall towards M...